cypress/e2e/tests/pages/fleet/gitrepo.spec.ts

```diff
@@ -87,6 +88,18 @@ describe('Git Repo', { testIsolation: 'off', tags: ['@fleet', '@adminUser'] }, (
       gitRepoCreatePage.gitAuthSelectOrCreate().createSSHAuth('test1', 'test1', 'KNOWN_HOSTS');
       gitRepoCreatePage.helmAuthSelectOrCreate().createBasicAuth('test', 'test');
       gitRepoCreatePage.setHelmRepoURLRegex(helmRepoURLRegex);
+      // #Percy tests
+      cy.get('i.checkbox-info.icon-info.icon-lg.v-popper--has-tooltip').realHover();
+      cy.wait(500);
```

Comment:
`wait` should be avoided unless estrictly necessary. You can use internal cypress timeouts or existing custom cypress commands (`waitForXXX`),.

Reply:
done